src/Service/ConfigurationService.php

```diff
@@ -124,22 +126,18 @@ public function getHmacLive()
      */
     public function getHmacKey()
     {
-        if ($this->getEnvironment()) {
-            return $this->getHmacLive();
-        }
-
-        return $this->getHmacTest();
+        return $this->systemConfigService->get(
+            self::BUNDLE_NAME . '.config.hmac' . ucfirst($this->getEnvironment())
+        );
     }
 
     /**
      * @return array|mixed|null
      */
     public function getApiKey()
     {
-        if ($this->getEnvironment()) {
-            return $this->getApiKeyLive();
-        }
-
-        return $this->getApiKeyTest();
+        return $this->systemConfigService->get(
+            self::BUNDLE_NAME . '.config.apiKey' . ucfirst($this->getEnvironment())
+        );
```

Comment:
The same here.

```suggestion
        ```suggestion
        if ($this->getEnvironment() === Environment::LIVE) {
            return $this-> getApiKeyLive();
        )
        
        return $this-> getApiKeyTest();
```
```